ajax/libs/blissfuljs/package.json

```diff
@@ -18,18 +18,13 @@
   "author": "Lea Verou",
   "license": "MIT",
   "homepage": "https://blissfuljs.com",
-  "autoupdate": {
-    "source": "git",
-    "target": "git://github.com/LeaVerou/bliss.git",
-    "fileMap": [
-      {
-        "basePath": "",
-        "files": [
-          "bliss.js",
-          "bliss.min.js",
-          "bliss.shy*.js"
-        ]
-      }
-    ]
-  }
+  "npmName": "blissfuljs",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "bliss?(.shy)?(.min).js"
```

Comment:
Let's be use a more flexible pattern `bliss!(._.)*` to just exclude possible file `bliss._.js`.